### PR TITLE
Xtina

### PR DESCRIPTION
**Pull request by chdastolfo**
It's my site, which consists of 3 pages! I would like to merge it with the master branch! Please?

### CODE REVIEW

**Comment by Kira-Pilot** [comment]
5 thumbs up

**Comment by dterana** [comment]
I'm not sure you have to remove the previous link in the index.html file.
Maybe we can discus about this point today.

NB: I love the otters :)

**Comment by chdastolfo** [comment]
Yeah I didn't mean to do that. My bad. I hope you can all forgive me.

On Wed, Jun 29, 2016 at 11:15 AM, David TERANA notifications@github.com
wrote:

> I'm not sure you have to remove the previous link in the index.html file.
> Maybe we can discus about this point today.
> 
> NB: I love the otters :)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/nw0428/SibSummer2016/pull/2#issuecomment-229388777,
> or mute the thread
> https://github.com/notifications/unsubscribe/AIrM1xJbUqBmT5-LkvUKrR1EyrloHV7bks5qQowWgaJpZM4JAu9T
> .

## 

Christina D'Astolfo, MPA
Greater Boston Program Coordinator
Raising A Reader MA

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <h1>Summer 2016 SIB Class!!!</h1>
   <div class="center">
     <ul>
-    	<li><a href="pages/david.html">David Zabner</a></li>
```
Comment (nw0428) [inline index.html:10]: Put this back!

**Comment by nw0428** [comment]
One comment then LGTM

**Comment by dterana** [comment]
It will be hard, but I will try harder.

**Comment by dmasters19** [comment]
These chickpeas are so bad

**Pull request merged**